src/ADPCMSoundLoader.js

```diff
@@ -56,7 +56,7 @@ ADPCMSoundLoader.prototype.load = function (url) {
 
             var samples = this.imaDecompress(this.extractChunk('data', stream), this.adpcmBlockSize);
 
-            // this line is the only place Tone is used here, should be possible to remove
+            // todo: this line is the only place Tone is used here, should be possible to remove
```

Comment:
This does not seem to have any effect on what the PR is about. This should probably be added in a separate PR

Just my opinion.